### ISSUE TITLE
custom wp_editor broken from videopress module

### ISSUE THREAD

**Issue by Lewiscowles1986** [Automattic/jetpack#4060]
#### What I expected
I Expected an existing website to continue to work with Jetpack installed

#### What happened instead
Instead the wp_editor seems to render empty and not work

#### Steps to reproduce the issue
Install Jetpack, activate (doesn't matter if you disable videopress module for some reason)
add a custom wp_editor somewhere after deactivating the built-in editor.

![videopress-tooltip](https://cloud.githubusercontent.com/assets/2605791/15761855/5f2524ca-2912-11e6-8b69-0865fe018f1a.png)
![videopress editor-view js](https://cloud.githubusercontent.com/assets/2605791/15761854/5f22fb5a-2912-11e6-9f1a-7ac6bf0d2a1f.png)

#### Current temp-fix

```php
    public function on_init() {
        \add_action( 'wp_print_scripts', array( $this, 'dequeue_scripts' ), 99 );
        \add_action( 'admin_notices', array( $this, 'dequeue_scripts' ), 99 );
    }
    public function dequeue_scripts() {
        \wp_dequeue_script( 'videopress-editor-view' );
    }
```

I'm currently fixing using this ( My Plugin is an object, hence the `public` and `array($this, 'method')` )

**Comment by jeherve** [comment]
Closing as duplicate of #3799

**Comment by jeherve** [comment]
@Lewiscowles1986 You can follow our discussion on a possible fix in #4044

**Comment by Lewiscowles1986** [comment]
Sorry
